tests/media/test_media_storage.py

```diff
@@ -294,7 +294,9 @@ def _req(
         self.assertEqual(
```

Comment:
Are there tests for all this new behavior somewhere? I don't see Complement tests

Reply:
I got a start on it here: https://github.com/matrix-org/complement/pull/641

I'm not sure how to configure this PR to actually run the test, though.

Reply:
Same branch name is usually the solution but not sure how it works for forks